app/assets/src/components/views/phylo_tree/PhyloTreeListView.jsx

```diff
@@ -28,7 +28,11 @@ class PhyloTreeListView extends React.Component {
   }
 
   getDefaultSelectedTreeId(urlParams, phyloTrees = []) {
-    return urlParams.treeId || (phyloTrees[0] || {}).id;
+    return (
+      urlParams.treeId ||
+      parseInt(window.sessionStorage.getItem("treeId")) ||
```

Comment:
did we all agree that window.sessionStorage is OK? is it supported on all the modern browsers

Reply:
Should be fine: https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage. We should do a more through css/js audit for issues in older browsers.